### ISSUE TITLE
Redesign Store to align with new brand

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#589]
[Figma designs](https://www.figma.com/file/ryjFGAj3HwVA5pIgVtVU5A/CNCF_Store?node-id=194%3A2018)

**Comment by KTan1226** [comment]
Hi Chris - thanks for the update. Do we have a timeline on updating the store? Could we aim for early 2023?

**Comment by cjyabraham** [comment]
I would hope to have it done before the end of this year, however, Megan (from Summit) is the one who will be implementing it and it's not clear right now how fast she can do it and where it lies in her priorities. I think that should become clear in the coming week or so.

**Comment by KTan1226** [comment]
Sounds good, we'll keep in touch on it. I also asked Drew and Amanda to provide us with a cost estimate for the new swag. Thanks!

**Comment by cjyabraham** [comment]
@thetwopct assigning to you to do a round of polish on what Megan has left us

**Comment by thetwopct** [comment]
Are you able to be more specific with what you're looking me to implement @cjyabraham?

**Comment by cjyabraham** [comment]
As discussed, try and find some reasonable middle-ground between the theme as Megan has it and the designs Gareth has drafted. Do the easy customizations and leave the details that would require too much work to achieve. It doesn't have to be perfect but around the same standard as other platforms we've customized such as the [Jobs Board](https://jobs.cncf.io/) and [Community site](https://community.cncf.io/).

**Comment by thetwopct** [comment]
[Store preview link](https://vmgqwe4e25m0a0r0-13008977.shopifypreview.com/) (valid for 48 hours from now)

### To do, waiting on exact info:
- Add more spacing to the menu nav on desktop, waiting on what spacing
- Front page slider text background to be changed, waiting on more info
- Revise footer, waiting on more info

### To do, nice to have:
- Move sliders arrows over the top of the slider, instead of below

### To do:
- [ ] Hover line on menu still seems 2 pixels too high
- [ ] Push the logo left a bit on mobile
- [ ] H2s on mobile on the FAQ seem kinda small. Maybe make them a bit bigger
- [ ] Remove hanging feature slider?

### To do, after theme is live:
- [ ] Attach custom cover image to each collection
- [ ] Add sub-title description to each collection 
- [ ] Remove Shop by Projects dropdown with a link to All Collections page
- [ ] Check all content - some content has weird text size - see Phippy & Friends Stress Toys Set
- [ ] Run performance checks

**Comment by cjyabraham** [comment]
[Performance checks](https://pagespeed.web.dev/report?url=https%3A%2F%2Fstore.cncf.io%2F&form_factor=desktop) produce this:
<img width="577" alt="Screen Shot 2022-12-08 at 10 31 55 AM" src="https://user-images.githubusercontent.com/463160/206537783-dd52aa67-a650-459c-8008-6d26fa206b10.png">

**Comment by thetwopct** [comment]
Nice

**Comment by cjyabraham** [comment]
Actually for mobile it wasn't as promising:
<img width="577" alt="Screen Shot 2022-12-08 at 10 34 06 AM" src="https://user-images.githubusercontent.com/463160/206538389-12d44161-0d29-4b39-8f2a-4152a313e9c0.png">
<img width="1168" alt="Screen Shot 2022-12-08 at 10 34 17 AM" src="https://user-images.githubusercontent.com/463160/206538381-3471f56f-26a1-4e86-8e75-c6de4cb5d2e5.png">

**Comment by cjyabraham** [comment]
Splitting out [outstanding design tasks](https://github.com/cncf/cncf.io/issues/635) and [outstanding copy tasks](https://github.com/cncf/cncf.io/issues/636).
